samples/winapi_drivelist/main.c

```diff
@@ -56,11 +65,30 @@ int main(void)
         debugPrint("%s\n", drive);
         while(*drive++);
     }
-    debugPrint("\ndone");
 
-    while(1) {
+    goto unmount_c;
+    goto unmount_e;
+
+    debugPrint("\nDone!");
+
+    while (1) {
         Sleep(2000);
     }
 
     return 0;
-}
+
+    unmount_c:
```

Comment:
This is the wrong order, `unmount_e` needs to come before `unmount_c, otherwise a failure when mounting E would cause a goto to `unmount_c`, which would unmount C, and subsequently it would try to unmount E, which was never mounted.